components/ImageGallery.js

```diff
@@ -0,0 +1,356 @@
+import React, { Component, PropTypes } from 'react';
+
+import {
+  ScrollView,
+  LayoutAnimation,
+  InteractionManager,
+} from 'react-native';
+
+import { connectStyle } from '@shoutem/theme';
+import { TimingDriver } from '@shoutem/animation';
+
+import Image from 'react-native-transformable-image';
```

Comment:
I wouldn't include it in the toolkit for now. We are currently using this component only in some situations (iOS gallery, and lightbox), but we are also using a different, completely native, component in the gallery on Android.